homeassistant/components/sleepiq.py

```diff
@@ -39,7 +39,7 @@
 DATA = None
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
+    vol.Required(DOMAIN): vol.Schema({
```

Comment:
From my testing, it looked like not having this meant that not having `sleepiq` in your config was ok too (but it shouldn't, username is required)